drivers/vulkan/vulkan_context.h

```diff
@@ -184,19 +185,13 @@ class VulkanContext {
 	int command_buffer_count = 1;
 
 	// Extensions.
+	static HashMap<CharString, bool> requested_instance_extensions;
```

Comment:
I wonder if there is ever a situation where you would want to create a local device with different extensions than the main device?

I guess not, because the extensions supported will be the same, so it would make more sense to request the extension for both and then check if it is available.

Reply:
I'm not sure what the use case is for creating a local device, we were already re-using the extension so I just kept it the same.